HW04/Alexey_Zalyotov/spec/my_module_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'my_module'
+
+class Array
+  include MyModule
+end
+
+describe '#my_each' do
+  subject { Array.new([1, 2, 3]) }
+  context 'when no block is given' do
```

Comment:
If you have this case you need to add and opposite as well 
```suggestion
  context 'when no block is given' do
  context 'when block is given' do
```

Reply:
Done